app/Console/Commands/Game.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Cache\Repository as CacheRepository;
+use Illuminate\Contracts\Config\Repository as ConfigRepository;
+
+class Game extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'game:test';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    private CacheRepository $cacheRepository;
+
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(CacheRepository $cacheRepository, ConfigRepository $configRepository)
+    {
+        parent::__construct();
+        $this->cacheRepository = $cacheRepository;
+        $this->configRepository = $configRepository;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+
+
+    public function handle()
+    {
+
+        $lastVisit = $this->cacheRepository->get('lastCommandVisit', 'never');
+        $diedStats = $this->cacheRepository->get('diedCount', 0);
+        $surviveStats = $this->cacheRepository->get('surviveCount', 0);
+
+
+        $this->info("Command called last time: {$lastVisit}");
+        $this->info("Died Statistic: {$diedStats}");
+        $this->info("Survive Statistic: {$surviveStats}");
+
+        $this->cacheRepository->set('lastCommandVisit', date(\DateTime::ATOM), 86400);
+
+
+        $executionCicle = True;
+        $count = 1;
+        $table = [];
+        $answer = 'survive';
+        $res = 0;
+
+        while ($executionCicle) {
+            $random = $random = random_int(0, 2);
```

Comment:
it should have random from 6 to emulate the revolver, and for statistics, I expect to receive the statistics, which number how many times was randomized.

Reply:
Change in random_int(1, 6), and add statistics for random numbers.